src/main/java/net/mcreator/element/ModElementTypeLoader.java

```diff
@@ -112,6 +112,8 @@ public static void loadModElements() {
 				new ModElementType<>("tag", 'j', BaseType.DATAPACK, RecipeType.NONE, TagGUI::new, Tag.class));
 		ModElementType.TOOL = register(
 				new ModElementType<>("tool", 't', BaseType.ITEM, RecipeType.ITEM, ToolGUI::new, Tool.class));
+		ModElementType.VILLAGERTRADE = register(
+				new ModElementType<>("villagertrade", '6', BaseType.VILLAGERTRADE, RecipeType.NONE, VillagerTradeGUI::new, VillagerTrade.class));
```

Comment:
You should not put random character here, we ran out of them so use no character here

Reply:
Wdym? What character?

Reply:
Klemen, in my own PR I started to use digits and you didn't say I shouldn't. I see no reason why Max shouldn't too.

Reply:
@Goldorion, I did not notice this, please use null for elements that did not have a shortcut before